fastlane/swift/Actions.swift

```diff
@@ -1,16 +1 @@
-// Actions.swift
-// Copyright (c) 2022 FastlaneTools
-
-// This autogenerated file will be overwritten or replaced when running "fastlane generate_swift"
-//
-//  ** NOTE **
-//  This file is provided by fastlane and WILL be overwritten in future updates
-//  If you want to add extra functionality to this project, create a new file in a
-//  new group so that it won't be marked for upgrade
-//
-
-import Foundation
-
-// Please don't remove the lines below
-// They are used to detect outdated files
-// FastlaneRunnerAPIVersion [0.9.56]
+import Foundation
```

Comment:
@Thieurom I ran the setup command and it is auto generated as-is, I believe it is okay to leave it like that because the same situation is also happening on [your PR](https://github.com/Thieurom/ic-surveys-ios/pull/34/files#diff-ae349eb754cb256923644af27128bf045dbdef352fe450c81c2991fa8155f2daL1-R1) 🙈